Cython/Debugger/libcython.py

```diff
@@ -117,6 +117,25 @@ def wrapper(self, string, *args, **kwargs):
 # Classes that represent the debug information
 # Don't rename the parameters of these classes, they come directly from the XML
 
+def simple_repr(self, renamed={}, skip=[], state=True):
+    import inspect
+    args = tuple(inspect.signature(self.__init__).parameters)
+    args = tuple(renamed[i] if i in renamed else i for i in args)
+    local = tuple(filter(lambda x: x not in args, vars(self).keys())) \
+        if state else ()
+
+    def equals(prefix, attrs):
+        for i in (j for j in attrs if j not in skip):
+            attr = repr(getattr(self, i))
+            yield prefix + i + " = " + attr.replace("\n", "\n\t\t")
+
+    return "".join((
+            self.__class__.__name__,
+            "(", ",".join(equals("\n\t\t", args)), "\n\t)",
+            *equals("\nself.", local)
+        ))
```

Comment:
This doesn't feel like a tuple, more like a list.
```suggestion
    return "".join([
            self.__class__.__name__,
            "(", ",".join(equals("\n\t\t", args)), "\n\t)",
            *equals("\nself.", local)
        ])
```